python_training/utils.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy.orm import sessionmaker
+
+
+def get_session(engine):
```

Comment:
This will work much better as a context manager.
In its current implementation, it's the user's responsibility to close the `session` every time, which is tricky (notice how you forgot to do it in `main.py`, for example).